### PR TITLE
[MAIN] [STRATCONN-3905] Klaviyo Actions order completed. 

### PR DESCRIPTION
**Pull request by AnkitSegment**
Added `Event Name` field in Order completed mapping actions. 

Screenshot 2024-08-07 at 12 11 18 PM

## Testing

_Include any additional information about the testing you have completed to
ensure your changes behave as expected. For a speedy review, please check
any of the tasks you completed below during your testing._

Screenshot 2024-08-07 at 12 35 22 PM

Screenshot 2024-08-07 at 12 34 45 PM

Screenshot 2024-08-08 at 10 22 14 AM

Screenshot 2024-08-08 at 5 37 43 PM (1)

- [X] Added [unit tests](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing) for new functionality
- [X] Tested end-to-end using the [local server](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing)
- [X] [Segmenters] Tested in the staging environment

### CODE REVIEW

**Review by varadarajan-tw — COMMENTED** [review]
Left one minor comment. Also, the test screenshot attached still shows Order Completed? 🤔

File: packages/destination-actions/src/destinations/klaviyo/orderCompleted/index.ts [hunk 1]
```diff
@@ -155,6 +161,10 @@ const action: ActionDefinition<Settings, Payload> = {
       throw new PayloadValidationError('One of External ID, Anonymous ID, Phone Number or Email is required.')
     }
 
+    if (!payload.event_name) {
```
Comment (varadarajan-tw) [inline packages/destination-actions/src/destinations/klaviyo/orderCompleted/index.ts:164]: Could you add a comment on why we are doing this?

Something like 'For backward comptability: Fallback to Order Completed if payload.event_name is undefined'

**Review by varadarajan-tw — APPROVED** [review]
LGTM!

**Comment by joe-ayoub-segment** [comment]
hi @AnkitSegment this PR has been deployed

**Pull request merged**